### PR TITLE
fix: a11y improvements

### PR DESCRIPTION
**Pull request by richard015ar**
Issue: https://github.com/pressbooks/pressbooks-multi-institution/issues/129

This PR addresses some of the suggestions mentioned in: https://github.com/pressbooks/pressbooks-multi-institution/issues/129.

### CODE REVIEW

File: resources/assets/styles/pressbooks-multi-institutions.css [hunk 1]
```diff
@@ -57,6 +57,14 @@ table.users, table.assign-books {
 	}
 }
 
+table.institutions-totals-table {
```
Comment (richard015ar) [inline resources/assets/styles/pressbooks-multi-institutions.css]: it is ok to create class names with `-`? I was unsure if I should use `_` o `-`

Comment (greatislander) [reply resources/assets/styles/pressbooks-multi-institutions.css]: Either is fine, I tend to prefer kebab case for this kind of situation.

File: resources/views/institutions/form.blade.php [hunk 2]
```diff
@@ -235,8 +253,8 @@ class="regular-text"
 						<input
 							name="book_limit"
 							id="book_limit"
-							type="number"
-							min="0"
+							inputmode="numeric" pattern="[0-9]*"
```
Comment (greatislander) [inline resources/views/institutions/form.blade.php]: This needs a `type="text"` to pick up the default input styling. Right now it looks a little funny.

Screenshot showing improperly styled input

File: resources/views/institutions/form.blade.php [hunk 3]
```diff
@@ -235,8 +253,8 @@ class="regular-text"
...
+							title="{{ __('Only numbers are allowed', 'pressbooks-multi-institution') }}"
```
Comment (greatislander) [inline resources/views/institutions/form.blade.php]: I would avoid using a `title` as this is not consistently supported. I'd suggest instead adding a visually hidden `<div>` with idea and referencing it in the `aria-describedby` attribute, e.g.

```
aria-describedby="book_limit-description book_limit-input-instruction"
```

Paired with:

```blade
<div class="screen-reader-text" id="book_limit-instruction">{{ __('Only numbers are allowed', 'pressbooks-multi-institution') }}</div>
```

**Review by greatislander — CHANGES_REQUESTED** [review]
Two suggested changes.

One other issue is that I realized that the button which dismisses the error notice is included in the announced message because I think WordPress inserts it dynamically after load just like the other content is inserted. This is what's announced when I try to create two institutions both called "Dal":

VoiceOver output which reads: Dismiss this notice. Error: the form is invalid. An institution with the name Dal already exists. Please use a different name.

I would suggest for these you could remove the `is-dismissable` class. WordPress adds `role="alert"` to them dynamically so creating a live region within the notice that doesn't include the dismiss button wouldn't work.

File: resources/views/assign/index.blade.php [hunk 4]
```diff
@@ -1,6 +1,17 @@
 @if (isset($result['message']))
-	<div id="message" role="status" class="updated {{ $result['success'] ? 'notice' : 'error' }} is-dismissible">
-		<p>{{ $result['message'] }}</p>
+	<div
+		id="message"
+		role="status"
```
Comment (greatislander) [inline resources/views/assign/index.blade.php]: This should be `role="alert"`. It also doesn't seem to be necessary as WordPress adds this dynamically/

**Comment by richard015ar** [comment]
Code LGTM

**Review by fdalcin — APPROVED** [review]
LGTM

**Pull request merged**